### ISSUE TITLE
failed when reinstall of sylius

### ISSUE THREAD

**Issue by ghanbari** [Sylius/Sylius#9028]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Sylius version   | 1.0.5

I try reinstall sylius by `sylius:install` command, but it fail in step 4.

error report: 
```
PHP Warning:  Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: require(/var/www/sylius/var/cache/de_/ContainerVpotf3m/getDebug_DumpListenerService.php): failed to open stream: No such file or directory in /var/www/sylius/var/cache/de_/ContainerVpotf3m/appDevDebugProjectContainer.php:2878
Stack trace:
#0 /var/www/sylius/var/cache/de_/ContainerVpotf3m/appDevDebugProjectContainer.php(2878): require()
#1 /var/www/sylius/var/cache/de_/ContainerVpotf3m/appDevDebugProjectContainer.php(3961): ContainerVpotf3m\appDevDebugProjectContainer->load('/var/www/sylius...')
#2 /var/www/sylius/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(229): ContainerVpotf3m\appDevDebugProjectContainer->ContainerVpotf3m\{closure}()
#3 /var/www/sylius/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php(61): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('console.command')
#4 /var/www/sylius/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher in /var/www/sylius/var/cache/de_/ContainerVpotf3m/appDevDebugProjectContainer.php on line 2878
PHP Fatal error:  ContainerVpotf3m\appDevDebugProjectContainer::load(): Failed opening required '/var/www/sylius/var/cache/de_/ContainerVpotf3m/getDebug_DumpListenerService.php' (include_path='.:/usr/share/php') in /var/www/sylius/var/cache/de_/ContainerVpotf3m/appDevDebugProjectContainer.php on line 2878

In appDevDebugProjectContainer.php line 2878:
                                                                                                                                                      
  Compile Error: ContainerVpotf3m\appDevDebugProjectContainer::load(): Failed opening required '/var/www/sylius/var/cache/de_/ContainerVpotf3m/getDe  
  bug_DumpListenerService.php' (include_path='.:/usr/share/php') 
```

I have not `_de` cache folder, and have not this env.

**Comment by pamil** [comment]
Could you post the list of installed packages (`composer show`) and PHP version you're using (`php -v`)?

**Comment by wadjeroudi** [comment]
Tried on sylius 1.0.5 with symfony 3.3.13, I don't have the problem, but have it with sf 3.4.2.

**Comment by ghanbari** [comment]
this is output of `composer show`:
```
behat/behat                                     v3.4.3  Scenario-oriented BDD framework for PHP 5.3
behat/gherkin                                   v4.5.1  Gherkin DSL parser for PHP 5.3
behat/mink                                      v1.7.1  Browser controller/emulator abstraction for PHP
behat/mink-browserkit-driver                    v1.3.2  Symfony2 BrowserKit driver for Mink framework
behat/mink-extension                            2.3.0   Mink extension for Behat
behat/mink-selenium2-driver                     v1.3.1  Selenium2 (WebDriver) driver for Mink framework
behat/transliterator                            v1.2.0  String transliterator
bossa/phpspec2-expect                           1.0.2   Helper that decorates any SUS with a phpspec lazy object wrapper
clue/stream-filter                              v1.4.0  A simple and modern approach to stream filtering in PHP
cocur/slugify                                   v3.0.1  Converts a string into a slug.
coduo/php-matcher                               2.3.0   PHP Matcher enables you to match values with patterns
coduo/php-to-string                             2.0.1  
composer/ca-bundle                              1.1.0   Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA b...
container-interop/container-interop             1.2.0   Promoting the interoperability of container objects (DIC, SL, etc.)
dama/doctrine-test-bundle                       v1.0.13 Symfony 2/3 bundle to isolate doctrine database tests and improve test performance
doctrine/annotations                            v1.6.0  Docblock Annotations Parser
doctrine/cache                                  v1.7.1  Caching library offering an object-oriented API for many cache backends
doctrine/collections                            v1.5.0  Collections Abstraction library
doctrine/common                                 v2.8.1  Common Library for Doctrine projects
doctrine/data-fixtures                          v1.3.0  Data Fixtures for all Doctrine Object Managers
doctrine/dbal                                   v2.6.3  Database Abstraction Layer
doctrine/doctrine-bundle                        1.8.1   Symfony DoctrineBundle
doctrine/doctrine-cache-bundle                  1.3.2   Symfony Bundle for Doctrine Cache
doctrine/doctrine-fixtures-bundle               v2.4.1  Symfony DoctrineFixturesBundle
doctrine/doctrine-migrations-bundle             v1.3.1  Symfony DoctrineMigrationsBundle
doctrine/inflector                              v1.2.0  Common String Manipulations with regard to casing and singular/plural rules.
doctrine/instantiator                           1.1.0   A small, lightweight utility to instantiate objects in PHP without invoking their constru...
doctrine/lexer                                  v1.0.1  Base library for a lexer that can be used in Top-Down, Recursive Descent Parsers.
doctrine/migrations                             v1.6.2  Database Schema migrations using Doctrine DBAL
doctrine/orm                                    v2.5.13 Object-Relational-Mapper for PHP
egulias/email-validator                         2.1.3   A library for validating emails against several RFCs
fig/link-util                                   1.0.0   Common utility implementations for HTTP links
friends-of-behat/context-service-extension      v0.3.0  Allows to declare and use contexts services in scenario scoped container.
friends-of-behat/cross-container-extension      v0.2.1  Makes possible to inject services and parameters from other containers.
friends-of-behat/performance-extension          v1.0.2  Accelerates Behat using features available only for newer PHP versions.
friends-of-behat/service-container-extension    v0.3.0  Allows to declare own services inside Behat container without writing an extension.
friends-of-behat/suite-settings-extension       v1.0.1  Allows to overwrite suites' default settings.
friends-of-behat/symfony-extension              v0.2.1  Integrates Behat with Symfony (both 2 and 3).
friends-of-behat/variadic-extension             v0.1.0  Variadic support for behat context arguments
friendsofsymfony/oauth-server-bundle            1.5.2   Symfony2 OAuth Server Bundle
friendsofsymfony/oauth2-php                     1.2.1   OAuth2 library
friendsofsymfony/rest-bundle                    2.3.0   This Bundle provides various tools to rapidly develop RESTful API's with Symfony
fzaninotto/faker                                v1.7.1  Faker is a PHP library that generates fake data for you.
gedmo/doctrine-extensions                       v2.4.31 Doctrine2 behavioral extensions
guzzlehttp/guzzle                               6.3.0   Guzzle is a PHP HTTP client library
guzzlehttp/promises                             v1.3.1  Guzzle promises library
guzzlehttp/psr7                                 1.4.2   PSR-7 message implementation that also provides common utility methods
hamcrest/hamcrest-php                           v2.0.0  This is the PHP port of Hamcrest Matchers
imagine/imagine                                 v0.7.1  Image processing for PHP 5.3
incenteev/composer-parameter-handler            v2.1.2  Composer script handling your ignored parameter file
instaclick/php-webdriver                        1.4.5   PHP WebDriver for Selenium 2
jdorn/sql-formatter                             v1.2.17 a PHP SQL highlighting library
jeremykendall/php-domain-parser                 3.0.0   Public Suffix List based URL parsing implemented in PHP.
jms/metadata                                    1.6.0   Class/method/property metadata management in PHP
jms/parser-lib                                  1.0.0   A library for easily creating recursive-descent parsers.
jms/serializer                                  1.10.0  Library for (de-)serializing data of any complexity; supports XML, JSON, and YAML.
jms/serializer-bundle                           2.3.1   Allows you to easily serialize, and deserialize data of any complexity
knplabs/gaufrette                               v0.4.0  PHP5 library that provides a filesystem abstraction layer
knplabs/knp-gaufrette-bundle                    0.3.0   Allows to easily use the Gaufrette library in a Symfony project
knplabs/knp-menu                                2.3.0   An object oriented menu library
knplabs/knp-menu-bundle                         v2.2.0  This bundle provides an integration of the KnpMenu library
lakion/api-test-case                            v1.2.0  Perfect PHPUnit TestCase for JSON/XML API TDD with Symfony.
lakion/mink-debug-extension                     v1.2.3  Debug extension for Behat
league/uri                                      4.2.2   URI manipulation library
liip/imagine-bundle                             1.9.1   This bundle provides an image manipulation abstraction toolkit for Symfony-based projects.
mockery/mockery                                 1.0     Mockery is a simple yet flexible PHP mock object framework for use in unit testing with P...
monolog/monolog                                 1.23.0  Sends your logs to files, sockets, inboxes, databases and various web services
myclabs/deep-copy                               1.7.0   Create deep copies (clones) of your objects
nelmio/alice                                    v2.3.4  Expressive fixtures generator
ocramius/package-versions                       1.2.0   Composer plugin that provides efficient querying for installed package versions (no runti...
ocramius/proxy-manager                          2.1.1   A library providing utilities to generate, instantiate and generally operate with Object ...
openlss/lib-array2xml                           0.0.10  Array2XML conversion library credit to lalit.org
pagerfanta/pagerfanta                           v1.0.5  Pagination for PHP 5.3
paragonie/random_compat                         v2.0.11 PHP 5.x polyfill for random_bytes() and random_int() from PHP 7
payum/iso4217                                   1.0.1   ISO 4217 PHP Library
payum/payum                                     1.4.3   Payum offers everything you need to work with payments. From simplest use cases to very a...
payum/payum-bundle                              2.2.1   Rich payment solutions for symfony2. Paypal, payex, authorize.net, be2bill, omnipay, recu...
php-http/guzzle6-adapter                        v1.1.1  Guzzle 6 HTTP Adapter
php-http/httplug                                v1.1.0  HTTPlug, the HTTP client abstraction for PHP
php-http/message                                1.6.0   HTTP Message related tools
php-http/message-factory                        v1.0.2  Factory interfaces for PSR-7 HTTP Message
php-http/promise                                v1.0.0  Promise used for asynchronous HTTP requests
phpcollection/phpcollection                     0.5.0   General-Purpose Collection Library for PHP
phpdocumentor/reflection-common                 1.0.1   Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock               4.2.0   With this component, a library can provide support for annotations via DocBlocks or other...
phpdocumentor/type-resolver                     0.4.0  
phpoption/phpoption                             1.5.0   Option Type for PHP
phpspec/php-diff                                v1.1.0  A comprehensive library for generating differences between two hashable objects (strings ...
phpspec/phpspec                                 3.4.3   Specification-oriented BDD framework for PHP 5.6+
phpspec/prophecy                                1.7.3   Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage                       4.0.8   Library that provides collection, processing, and rendering functionality for PHP code co...
phpunit/php-file-iterator                       1.4.5   FilterIterator implementation that filters files based on a list of suffixes.
phpunit/php-text-template                       1.2.1   Simple template engine.
phpunit/php-timer                               1.0.9   Utility class for timing
phpunit/php-token-stream                        2.0.2   Wrapper around PHP's tokenizer extension.
phpunit/phpunit                                 5.7.25  The PHP Unit Testing framework.
phpunit/phpunit-mock-objects                    3.4.4   Mock Object library for PHPUnit
polishsymfonycommunity/symfony-mocker-container v1.0.2  Provides base Symfony dependency injection container enabling service mocking.
psr/cache                                       1.0.1   Common interface for caching libraries
psr/container                                   1.0.0   Common Container Interface (PHP FIG PSR-11)
psr/http-message                                1.0.1   Common interface for HTTP messages
psr/link                                        1.0.0   Common interfaces for HTTP links
psr/log                                         1.0.2   Common interface for logging libraries
psr/simple-cache                                1.0.0   Common interfaces for simple caching
ramsey/uuid                                     3.7.1   Formerly rhumsaa/uuid. A PHP 5.4+ library for generating RFC 4122 version 1, 3, 4, and 5 ...
se/selenium-server-standalone                   v2.53.1 Composer distribution of Selenium Server Standalone, the browser automation framework. Ad...
sebastian/code-unit-reverse-lookup              1.0.1   Looks up which function or method a line of code belongs to
sebastian/comparator                            1.2.4   Provides the functionality to compare PHP values for equality
sebastian/diff                                  1.4.3   Diff implementation
sebastian/environment                           2.0.0   Provides functionality to handle HHVM/PHP environments
sebastian/exporter                              2.0.0   Provides the functionality to export PHP variables for visualization
sebastian/global-state                          1.1.1   Snapshotting of global state
sebastian/object-enumerator                     2.0.1   Traverses array structures and object graphs to enumerate all referenced objects
sebastian/recursion-context                     2.0.0   Provides functionality to recursively process PHP variables
sebastian/resource-operations                   1.0.0   Provides a list of PHP built-in functions that operate on resources
sebastian/version                               2.0.1   Library that helps with managing the version number of Git-hosted PHP projects
sensio/distribution-bundle                      v5.0.21 Base bundle for Symfony Distributions
sensio/generator-bundle                         v3.1.7  This bundle generates code for you
sensiolabs/security-checker                     v4.1.6  A security checker for your composer.lock
sonata-project/block-bundle                     3.8.0   Symfony SonataBlockBundle
sonata-project/cache                            2.0.1   Cache library
sonata-project/core-bundle                      3.7.1   Symfony SonataCoreBundle
sonata-project/datagrid-bundle                  2.3.0   Symfony SonataDatagridBundle
sonata-project/intl-bundle                      2.4.0   Symfony SonataIntlBundle
stof/doctrine-extensions-bundle                 v1.2.2  Integration of the gedmo/doctrine-extensions with Symfony2
stripe/stripe-php                               v4.13.0 Stripe PHP Library
swiftmailer/swiftmailer                         v6.0.2  Swiftmailer, free feature-rich PHP mailer
sylius-labs/association-hydrator                v1.0.0  Doctrine ORM hydration performance optimization made easier.
sylius/sylius                                   v1.0.5  E-Commerce platform for PHP, based on Symfony framework.
symfony/monolog-bundle                          v3.1.2  Symfony MonologBundle
symfony/polyfill-apcu                           v1.6.0  Symfony polyfill backporting apcu_* functions to lower PHP versions
symfony/polyfill-iconv                          v1.6.0  Symfony polyfill for the Iconv extension
symfony/polyfill-intl-icu                       v1.6.0  Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring                       v1.6.0  Symfony polyfill for the Mbstring extension
symfony/polyfill-php56                          v1.6.0  Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70                          v1.6.0  Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-util                           v1.6.0  Symfony utilities for portability of PHP codes
symfony/swiftmailer-bundle                      v3.1.6  Symfony SwiftmailerBundle
symfony/symfony                                 v3.4.1  The Symfony PHP framework
twig/extensions                                 v1.5.1  Common additional features for Twig that do not directly belong in core
twig/twig                                       v2.4.4  Twig, the flexible, fast, and secure template language for PHP
webmozart/assert                                1.2.0   Assertions to validate method input/output with nice error messages.
white-october/pagerfanta-bundle                 v1.1.1  Bundle to use Pagerfanta with Symfony2
willdurand/hateoas                              2.11.0  A PHP library to support implementing representations for HATEOAS REST web services
willdurand/hateoas-bundle                       1.3.0   Integration of Hateoas into Symfony2.
willdurand/jsonp-callback-validator             v1.1.0  JSONP callback validator.
willdurand/negotiation                          v2.3.1  Content Negotiation tools for PHP provided as a standalone library.
winzou/state-machine                            0.3.2   A very lightweight yet powerful PHP state machine
winzou/state-machine-bundle                     v0.3.0  Bundle for the very lightweight yet powerful PHP state machine
zendframework/zend-code                         3.3.0   provides facilities to generate arbitrary code using an object oriented interface
zendframework/zend-eventmanager                 3.2.0   Trigger and listen to events within a PHP application
zendframework/zend-hydrator                     2.3.1  
zendframework/zend-stdlib                       3.1.0
```

**Comment by pamil** [comment]
Fixed by #9075 and will be released as v1.0.7.
